contrib/babelfishpg_tsql/src/backend_parser/gram-tsql-rule.y

```diff
@@ -4338,11 +4338,20 @@ tsql_IsolationLevelStr:
 				}
 			| REPEATABLE READ
 				{
-					TSQLInstrumentation(INSTR_UNSUPPORTED_TSQL_ISOLATION_LEVEL_REPEATABLE_READ);
-					ereport(ERROR,
+					if (strcmp(pltsql_repeatable_read_isolation, "pg_isolation") == 0)
```

Comment:
have we got feedback from PM about name? let us send a mail to ask with possible alternatives. pg_isolation looks good to me.